pkg/providers/aws/provider_redis_test.go

```diff
@@ -45,6 +52,19 @@ type mockStsClient struct {
 	stsiface.STSAPI
 }
 
+func buildTestSchemeRedis() (*runtime.Scheme, error) {
+	scheme := apimachinery.NewScheme()
+	err := croApis.AddToScheme(scheme)
+	err = openshiftConfv1.AddToScheme(scheme)
+	err = corev1.AddToScheme(scheme)
+	err = cloudCredentialApis.AddToScheme(scheme)
+	err = monitoringv1.AddToScheme(scheme)
+	if err != nil {
+		return nil, err
+	}
+	return scheme, nil
+}
```

Comment:
Is this scheme any different to the postgres one? If not, we could also just export this test scheme and use it across both files

Reply:
We could refactor to have a scheme that works for all providers, but it might be outside the scope of this particular PR, maybe keep it in the back of our minds to deal with at a later stage when we have time to address all the technical debt :laughing: 

Until then, I'd prefer to have a scheme specific to each provider.